ci/Dockerfile

```diff
@@ -78,10 +78,7 @@ RUN printf "\n\nADDING USER $USER_NAME TO SUDOERS - DEV ENV ONLY!\n\n" >&2 && \
     echo "$USER_NAME:$USER_PASSWORD" | chpasswd && \
     echo "$USER_NAME ALL=(ALL) NOPASSWD:ALL" >> /etc/sudoers
 
-# install scargo from spyro rep (temporarly before pip repo will be available)
-COPY requirements.txt requirements.txt
-RUN pip3 install -r requirements.txt && \
-    rm requirements.txt
+RUN pip3 install scargo[dev,doc]
```

Comment:
should be do it in that way - this is development only environment and we we shall not install scargo from pip here and we preparing all dependencies and testing them locally
I will keep the old req install

Reply:
True, I will revert this.